docs/Processes.md

```diff
@@ -221,8 +229,63 @@ the gateway api. Use .api.p“.gw.\*” for more details.
 | .gw.asyncexecjpt\[query; servertypes; joinfunction; postback; timeout\] | Execute the specified query against the required list of servers. Use the specified join function to aggregate the results. If the postback function is not set, the client must block and wait for the results. If it is set, the result will be wrapped in the specified postback function and returned asynchronously to the client. The query will be timed out if the timeout value is exceeded. |
 
 
-For the purposes of demonstration, assume that the queries must be run
-across an RDB and HDB process, and the gateway has one RDB and two HDB
+### Client Call Examples
+
+Here are a couple of examples for using client calls via a handle to the gateway process. 
+It is important to note if asynchronous calls are to be made, the .gw.synccallsallowed 
+variable needs to be set to 0b before running any async calls.
+To do this use
+`q) h(".gw.synccallsallowed:0b")`.
```

Comment:
Don't think we should encourage a client to do this. This variable should be controlled by config on startup. I think it is enough to mention that it will need to be set to false to run async.